modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -0,0 +1,98 @@
+= Add Behavioral Headers to Simulated API Calls
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can use behavioral headers to change the behavior of the mocking service.
+
+
+.Use behavioral headers where you would normally specify other types of header in the API Console
+image::apid-behavioral-headers.png[]
+
+
+MS2-Delay:: Indicates how many milliseconds the request should take to answer. You might want to use this header to simulate load on your server or latency in a network.
++
+You can specify a value for this header as an integer number or as a range of integer numbers. If you specify a range, the value must have the form `min-max`. For example, given the value `500-1000`, the request will take a random time between 500 and 1000 milliseconds.
++
+[NOTE]
+====
+It is possible for the delay to be longer than a single integer value or the high end of a range, depending on the latency of your network.
+====
+
+MS2-Error-Level:: Indicates which range of codes will be considered errors. You can specify one of the following three case-sensitive values:
++
+* `REQUEST_ONLY`: Only 4XX codes are considered to be errors.
+* `SERVER_ONLY`: Only 5XX codes are considered to be errors.
+* `ALL`: Both 4XX and 5XX codes are considered to be error. This is the default value.
+
+MS2-Error-Rate:: Indicates the probability of getting an error status code as a response. The value for this header must be a decimal number between 0 and 1.
++
+This header works together with MS2-Status-Code and MS2-Error-Level, but not both at the same time.
++
+* If you use this header together with MS2-Status-Code and the mocking service (based on the probability) determines that an error code should be returned, the mocking service uses the error code that is specified by MS2-Status-Code.
```

Comment:
I believe we avoid "should". Can it be reworded to avoid that term? 

Same comment for all the "should be returned".

Reply:
I replace the bullet points with a table that does not use "should".